benchmarks/viscoelastic_beam_modified/10km_opentopbot.prm

```diff
@@ -0,0 +1,92 @@
+# This parameter file modifies the 2D benchmark 'viscoelastic_bending_beam'.
+# It has been modified to model a much larger beam, and to have an open top
+# and bottom boundary to allow the inelastic, low viscosity fluid surrounding the beam to 
+# be more easily displaced while the beam flexes.
+#
+# The benchmark examines bending of a viscoelastic beam surrounded by a less dense
+# and viscous (inelastic) fluid. Gravitational forces drive initial bending 
+# (elastic strain) of the beam. The analytic solution is from Geodynamics by 
+# Turcotte and Schubert. Models have been tested for deflections that are no greater 
```

Comment:
Can you point to the specific section of the book (+ figure number & equation numbers, etc) for this benchmark?